src/table/styles.js

```diff
@@ -77,29 +67,25 @@ export const StyledTableBody = styled(TableBody, {
 
 export const StyledBodyRow = styled(TableRow, {
   shouldForwardProp: (prop) => prop !== 'bodyCellStyle',
-})(({ hover, bodyCellStyle }) =>
-  hover
-    ? {
-        '&&:hover': {
-          '& td:not(.selected), th:not(.selected)': {
-            backgroundColor: bodyCellStyle.hoverBackgroundColor,
-            color: bodyCellStyle.hoverFontColor,
-          },
-        },
-      }
-    : {}
-);
+})(({ hover, bodyCellStyle }) => ({
+  '&&:hover': {
+    '& td:not(.selected), th:not(.selected)': {
+      backgroundColor: hover && bodyCellStyle.hoverBackgroundColor,
+      color: hover && bodyCellStyle.hoverFontColor,
+    },
+  },
+  '& :last-child': {
+    boxShadow: `inset 1px -1px 0 ${bodyCellStyle.borderColor}, inset -1px 0 0 ${bodyCellStyle.borderColor}`,
```

Comment:
So this is so that the last cell looks correct? I think a comment here would clarify

Reply:
same for the other ones

Reply:
sure, gonna add a comment.

Reply:
Done